src/amo/sagas/reviews.js

```diff
@@ -235,9 +253,25 @@ function* manageAddonReview(
     );
 
     yield put(setReview(reviewFromResponse));
+
+    if (savingRating) {
+      yield put(flashReviewMessage(SAVED_RATING));
+    }
+    if (savingReview) {
+      yield put(flashReviewMessage(SAVED_REVIEW));
+    }
+
+    // Make the message disappear after some time.
+    yield new Promise((resolve) =>
+      setTimeout(resolve, FLASH_SAVED_MESSAGE_DURATION),
+    );
```

Comment:
Does that block the saga for a certain `FLASH_SAVED_MESSAGE_DURATION`? 🤔

Reply:
I was about to say that given the non-trivial code required to write test cases, that might not be a good idea, but then I checked the redux-saga doc and this seems a rather used pattern since there is a dedicated effect for that: https://redux-saga.js.org/docs/api/#delayms-val

Reply:
Oh, good call on `delay()`. It looks like that [implements cancellation](https://github.com/redux-saga/redux-saga/blob/master/packages/delay-p/src/index.js#L10) which seems important. I'll try to use `delay()` instead.